internal/manager/task_autotag.go

```diff
@@ -516,12 +517,19 @@ func (t *autoTagFilesTask) getCount(ctx context.Context) (int, error) {
 
 	imageCount := imageResults.Count
 
-	_, galleryCount, err := r.Gallery.Query(ctx, t.makeGalleryFilter(), findFilter)
+	galleryCount, err := r.Gallery.Query(ctx, models.GalleryQueryOptions{
```

Comment:
Only related to this method definition change: https://github.com/stashapp/stash/pull/4423/files?diff=split&w=1#r1443579818
No change in behavior expected.